src/main/java/io/confluent/idesidecar/restapi/kafkarest/controllers/RelationshipUtil.java

```diff
@@ -0,0 +1,118 @@
+package io.confluent.idesidecar.restapi.kafkarest.controllers;
+
+import io.confluent.idesidecar.restapi.kafkarest.model.Relationship;
+import io.confluent.idesidecar.restapi.util.Crn;
+import org.eclipse.microprofile.config.ConfigProvider;
+
+/**
+ * Utility class for generating {@code related} links for Kafka resources.
+ */
+public final class RelationshipUtil {
+
+  private RelationshipUtil() {
+  }
+
+  private static final String CRN_AUTHORITY = "";
```

Comment:
Why is the authority blank here? That's not a valid CRN (or even resource name)

Reply:
I inferred it from examining an API response from confluent-local's kafka-rest server. Taking a closer look at confluentinc/kafka-rest, I see that they've defaulted the Crn authority to an empty string as well.

https://github.com/confluentinc/kafka-rest/blob/b7b1bba059e878dd28d2dbc5c51d25d3d747bd16/kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java#L413-L415

Reply:
Removed this.